### PR TITLE
add throttling 

### PR DESCRIPTION
**Pull request by danielnaumau**
Instead of creating own implementation, I just decided to reuse library

### CODE REVIEW

File: traveltimepy/sdk.py [hunk 1]
```diff
@@ -48,10 +48,19 @@
 
 class TravelTimeSdk:
 
-    def __init__(self, app_id: str, api_key: str, limit_per_host: int = 2, host: str = 'api.traveltimeapp.com') -> None:
+    def __init__(
+        self,
+        app_id: str,
+        api_key: str,
+        limit_per_host: int = 2,
+        rate_limit: int = 30,
```
Comment (MockusTravelTime) [inline traveltimepy/sdk.py:56]: perhaps 60 should be default to match plans we are selling?

**Comment by MockusTravelTime** [comment]
overall lgtm, but  lacks one part: if you send time map request with 10 departure searches this implementation would only count as 1 hit while it should be 10

**Comment by danielnaumau** [comment]
> overall lgtm, but lacks one part: if you send time map request with 10 departure searches this implementation would only count as 1 hit while it should be 10

Not really, it will count as 10

rate_limit = AsyncLimiter(sdk_params.rate_limit // window_size, sdk_params.time_window)

This method is responsible for controlling how many requests I can send at the same time and this window size is the amount of searches in one request. So if we have 20 searches and the rate limit is 10 searches per minute, then this sdk will send only one request with 10 searches in one minute. 

But there is another problem. As far as I remember we use travelTime too when we count hits, so should I take it into the consideration?

File: traveltimepy/sdk.py (hunk 1 above)
Comment (danielnaumau) [reply traveltimepy/sdk.py:56]: sure, changed to 60

**Pull request merged**